content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+    export const PATH: string = "/consumers/:section/sales/:saleId/reviews";
+    export const ENCRYPTED: Fetcher.IEncrypted = {
+        request: false,
+        response: false,
+    };
+
+    export function path(section: string, saleId: string): string
+    {
+        return `/consumers/${section}/sales/${saleId}/reviews`;
+    }
+    export const stringify = createStringifier<Input>();
+}
+
+/**
+ * Update an inquiry.
+ * 
+ * Update ordinary inquiry article. However, it would not modify the content reocrd
+ * {@link ISaleInquiry.IContent}, but be accumulated into the {@link ISaleInquiry.contents}. 
+ * Therefore, all of the poeple can read how the content has been changed.
+ * 
+ * @param connection connection Information of the remote HTTP(s) server with headers (+encryption password)
+ * @param request Instance of the Express.Request
+ * @param section Code of the target section
+ * @param saleId ID of the target sale
+ * @param id ID of the target article to be updated
+ * @param input New content to be overwritten
+ * @return The newly created content record
+ * @throw 400 bad request error when type of the input data is not valid
+ * @throw 401 unauthorized error when you've not logged in yet
+ * @throw 403 forbidden error when the article is not yours
+ * 
+ * @controller ConsumerSaleReviewsController.update()
+ * @path PUT /consumers/:section/sales/:saleId/reviews/:id
+ * @nestia Generated by Nestia - https://github.com/samchon/nestia
+ */
+export function update
+    (
+        connection: IConnection,
+        section: string,
+        saleId: string,
+        id: number,
+        input: Primitive<update.Input>
+    ): Promise<update.Output>
+{
+    return Fetcher.fetch
+    (
+        connection,
+        update.ENCRYPTED,
+        update.METHOD,
+        update.path(section, saleId, id),
+        input,
+        update.stringify
+    );
+}
+export namespace update
+{
+    export type Input = Primitive<ISaleReview.IStore>;
+    export type Output = Primitive<ISaleInquiry<ISaleReview.IContent>>;
+
+    export const METHOD = "PUT" as const;
+    export const PATH: string = "/consumers/:section/sales/:saleId/reviews/:id";
+    export const ENCRYPTED: Fetcher.IEncrypted = {
+        request: false,
+        response: false,
+    };
+
+    export function path(section: string, saleId: string, id: number): string
+    {
+        return `/consumers/${section}/sales/${saleId}/reviews/${id}`;
+    }
+    export const stringify = createStringifier<Input>();
+}
+```
+
+
+
+
+#### Swagger
+Building `Swagger` is also possible and even much powerful.
+
+Although I think the `Swagger` is a typical tool that torturing the client developers, the `nestia` also can build the `swagger.json` file. Even `Swagger` generator of the `nestia` is much powerful and convenient than the `@nestjs/swagger`. It doesn't require any type of the swagger comment or DTO decorator function and just using the pure interface type as DTO is possible.
```

Comment:
Remove the first person and negatively charged language here. The docs should remain impartial and read as technical documents, not a personal anecdote